selfdrive/car/toyota/carstate.py

```diff
@@ -94,6 +97,8 @@ def __init__(self, CP):
     self.shifter_values = self.can_define.dv["GEAR_PACKET"]['GEAR']
     self.left_blinker_on = 0
     self.right_blinker_on = 0
+    self.offset = 0.
+    self.is_offset_set = False
```

Comment:
not used

Reply:
PEP8: It's generally considered a good practice to explicitly define/declare all your attributes in the __init__ function, this way you have a quick list of all used (or unused) attributes in the class.

However, I'll remove these as OP doesn't strictly adhere to PEP8 ;)